src/Setup/Setup.vcxproj

```diff
@@ -29,7 +29,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
     <UseDebugLibraries>false</UseDebugLibraries>
-    <PlatformToolset>v120</PlatformToolset>
+    <PlatformToolset>v120_xp</PlatformToolset>
```

Comment:
Are comments allowed in here? Someone in the future might not realize that we need XP compatibility at this level in order to give your nice error message.